src/main/java/cynergy/CynergyKimFeed.java

```diff
@@ -0,0 +1,1085 @@
+package cynergy;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import javax.sql.DataSource;
+
+import org.apache.commons.dbcp.BasicDataSource;
+import org.apache.commons.lang.StringUtils;
+import org.apache.log4j.PropertyConfigurator;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.joda.time.DateTime;
+import org.joda.time.MutableDateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.rowset.SqlRowSet;
+
+import util.SimpleCommandLineParser;
+
+public class CynergyKimFeed {
+
+    private static final Logger LOG = LogManager.getLogger(CynergyKimFeed.class);
+
+	// Helper constant for inserting blank address rows.
```

Comment:
would it be worth while to do a source format of this class on this commit?  It should make future reviews easier.

Reply:
By "source format" are you referring to just the whitespace usage, or are you referring to other formatting like comments, line breaks, use of curly braces, etc.?

I would prefer to keep the changes in this PR as minimal as possible, since the CynergyKimFeed class will likely get completely replaced by DAO classes during the future refactoring efforts.

Reply:
I was referring to the eclipse function selecting the source of a java class, and then right clicking and choosing source, then format .... but I am fine with leaving as if the future refactor will be mostly a a replacement by DAOs

Reply:
It looks like the "source: format" feature would add way more changes than what I would have intended for an almost-drop-in setup of the existing class, even if they're only minor changes. If it's okay with you, I would prefer to wait on the formatting cleanup until the follow-up user story, which may totally replace the CynergyKimFeed class with some better DAOs anyway.

Reply:
OK, I am fine with proceeding